src/components/InfoPanel.component.js

```diff
@@ -57,6 +69,22 @@ class InfoPanel extends Component {
 
     return (
       <div className="info-panel">
+        <a class="btn btn-primary a-btn-slide-text">
+          <span class="glyphicon glyphicon-edit" onClick={this.onClickEdit} />
+          <span>
+            <strong>Edit</strong>
```

Comment:
The `strong` tag shouldn't be used since it's a styling related HTML tag. We should use css to style it, not HTML